requirements/edx-sandbox/base.in

```diff
@@ -16,14 +15,15 @@
 -c ../constraints.txt
 
 -r shared.txt                       # Dependencies in common with LMS and Studio
-chem                                # A helper library for chemistry calculations
+chem
 matplotlib==2.2.4                   # 2D plotting library
+networkx==2.2                       # Utilities for creating, manipulating, and studying network graphs
 numpy==1.16.5                       # Numeric array processing utilities; used by scipy
+openedx-calc
 pyparsing==2.2.0                    # Python Parsing module
-random2==1.0.1                      # Implementation of random module that works identically under Python 2 and 3
+random2                             # Implementation of random module that works identically under Python 2 and 3
 scipy==1.2.1                        # Math, science, and engineering library
-sympy==1.4                          # Symbolic math library
-git+https://github.com/edx/openedx-calc.git@e9b698c85ad1152002bc0868f475f153dce88952#egg=calc==0.4
+sympy==1.6.2                         # Symbolic math library
```

Comment:
The sympy upgrade seems like something we'd want to postpone until we're also upgrading Python and are preparing to look for code breakage anyway.

Reply:
We already did it with the python 2 upgrade.  This just updates base.in to match what's in prod.  I think moving to version specific files like you suggested will be clearer.